packages/firestore/test/unit/local/indexeddb_persistence.test.ts

```diff
@@ -1144,7 +1144,7 @@ describe('IndexedDb: allowTabSynchronization', () => {
       'clientA',
       /* multiClient= */ false,
       async db => {
-        db.injectFailures = true;
+        db.injectFailures = { updateClientMetadataAndTryBecomePrimary: true };
```

Comment:
Are you planning on making the other clients have the same transaction names? (Obviously they won't have the multitab related transactions, but how will this failure mechanism work anywhere else?)

Reply:
Yes. I am also planning to clean up our names before this happens.